test/json/test-functions.json

```diff
@@ -0,0 +1,259 @@
+{
+  "integer": [
+    { "src": "hello {4.2 :integer}", "exp": "hello 4" },
+    { "src": "hello {-4.20 :integer}", "exp": "hello -4" },
+    { "src": "hello {0.42e+1 :integer}", "exp": "hello 4" },
+    {
+      "src": ".match {$foo :integer} one {{one}} * {{other}}",
```

Comment:
when writing plural tests, I often find it a good idea to build out the variant matrix. Yes, `en-US` (or `und` for that matter) won't touch the `two` or `few` or whatever keyword. But it can help catch errors if one of the other keywords triggers incorrectly. It also allows the test itself to be recycled for other locales.